src/schemata/definitions/workflow_template/action_objects/getResources.yml

```diff
@@ -0,0 +1,46 @@
+---
+type: object
+title: Query Resources
```

Comment:
Let's align the title with the file name. I guess I prefer _query_ over _get_.... 🤔

Reply:
I went with Get Resources, I feel `Query` is a more generic term and we already have update resources.